website/cue/reference/components/aws_new_sdk.cue

```diff
@@ -0,0 +1,188 @@
+package metadata
+
+components: _aws_new_sdk: {
+	configuration: {
```

Comment:
It seems like only the `auth` configuration differs from `components: _aws_sdk`. Could we maybe inherit the other config and just define `auth` here?

Like:

```
components: _aws_new_sdk: {
  configuration: {
    auth: ...
    endpoint: components._aws_sdk.configuration.endpoint,
}
```

etc.